SmartDeviceLinkTests/DevAPISpecs/SDLFileManagerSpec.m

```diff
@@ -875,7 +875,7 @@ @implementation FileManagerSpecHelper
                     [testFileManager uploadArtworks:testArtworks progressHandler:^BOOL(NSString * _Nonnull artworkName, float uploadPercentage, NSError * _Nullable error) {
                         artworksDone++;
                         expect(artworkName).to(equal(expectedArtworkNames[artworksDone - 1]));
-                        expect(uploadPercentage).to(beCloseTo((float)artworksDone / 200.0));
+                        expect(uploadPercentage).to(beCloseTo((float)artworksDone / 200.0).within(0.01));
```

Comment:
What was the reason for this change?

Reply:
For whatever reason this test failed on any setup for me. I don’t recall the exact delta but it was higher that 0.001 (which seems to be default for “boCloseTo”)